NetKAN/World-Airforce-Insignia.netkan

```diff
@@ -0,0 +1,11 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "World-Airforce-Insignia",
+    "$kref"        : "#/ckan/github/chaseAEd/World-Airforce-Insignia",
+     "name"         : "World Airforce Insignia",
+    "abstract"     : "A simple mod that adds a range of current, or historic airforce roundels to add to your planes.",
+    "license"      : "MIT",
+    "ksp_version"  : "1.2.99",
+    "resources": {
+        "homepage": "https://github.com/chaseAEd/World-Airforce-Insignia"
+    }
```

Comment:
This resource block is unnecessary, as this will get automatically added by the github inflation process.

Also, you're missing the closing 

    }

To match the "{" at the start of the file.